src/modules/tests/test-summary/test-summary.selector.ts

```diff
@@ -0,0 +1,29 @@
+import { TestSummary } from '@dvsa/mes-test-schema/categories/B';
+
+export const getRouteNumber = (testSummary: TestSummary): number => testSummary.routeNumber;
+export const getCandidateDescription = (testSummary: TestSummary): string => testSummary.candidateDescription;
+export const getAdditionalInformation = (testSummary: TestSummary): string => testSummary.additionalInformation;
+export const isIdentificationLicense = (testSummary: TestSummary): boolean => {
```

Comment:
You could simplify this by converting the return in block on the right of the arrow to an expression. Also, no need for `|| false`, the equality expression does the same thing on its own. Same thing for quite a few of these selectors.